requirements-tests.txt

```diff
@@ -10,5 +10,3 @@ pygments==2.1.3
 codecov==2.0.5
 flake8==2.6.2
 coverage==4.2
-numpy==1.10.4
-matplotlib==1.5.1
```

Comment:
I don't think it's worth it on nightly or 3.6 dev. The install time is ugly on travis